charts/openshift-metering/templates/presto/presto-worker-config.yaml

```diff
@@ -13,12 +13,29 @@ data:
     node.environment=production
 
   config.properties: |
+    coordinator=false
+    node-scheduler.include-coordinator={{ .Values.presto.spec.presto.config.nodeSchedulerIncludeCoordinator }}
+    discovery-server.enabled=true
+
+    {{- if .Values.presto.spec.presto.config.useTLS }}
+    http-server.http.enabled=false
+    http-server.https.enabled=true
+    http-server.https.port=8080
+    http-server.https.keystore.path=/opt/presto/tls/keystore.jks
+    http-server.https.keystore.key=password
+    node.internal-address=presto
```

Comment:
This isn't correct, each worker will have a different address, this is the service name.

Reply:
Unfortunately, A records are not created for pod hostnames though, so we may need to create a headless service to make this work,

Reply:
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-hostname-and-subdomain-fields has details on this. I think we want to set subdomain + create headless service so we can get an A record per presto pod. Then we can set `node.internal-address=FQDN`

Reply:
Update: we've decided to go with hostnames using statefulsets. So this can use `FQDN` instead.